components/header/menu/index.js

```diff
@@ -0,0 +1,69 @@
+import React, { useState } from 'react';
+
+import { TOOLS } from '../constants';
+import cx from 'classnames';
+import Icon from 'components/icon';
+
+const HeaderMenu = () => {
```

Comment:
When the menu is closed, the menu is still accessible from the keyboard. You can easily prevent that by adding the `inert` attribute: https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/inert

React doesn't currently support the attribute so you'll have to use a workaround: https://github.com/facebook/react/issues/17157